<ui version="4.0" >
 <class>ImdbProcMain</class>
 <widget class="QMainWindow" name="ImdbProcMain" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>467</width>
    <height>253</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QGridLayout" >
    <property name="margin" >
     <number>9</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item rowspan="2" row="2" column="0" >
     <widget class="QGroupBox" name="optionsBox" >
      <property name="title" >
       <string>Options</string>
      </property>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="0" column="0" >
        <widget class="QLabel" name="minMoviesLabel" >
         <property name="text" >
          <string>Minumum number of movies:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" >
        <widget class="QSpinBox" name="minMovies" >
         <property name="minimum" >
          <number>1</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="1" >
     <spacer>
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" >
       <size>
        <width>51</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="0" colspan="3" >
     <widget class="QGroupBox" name="inputBox" >
      <property name="title" >
       <string>Input</string>
      </property>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="0" column="1" >
        <widget class="QPushButton" name="inputBrowse" >
         <property name="text" >
          <string>&amp;Browse...</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" >
        <widget class="QLineEdit" name="input" />
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0" colspan="3" >
     <widget class="QGroupBox" name="outputBox" >
      <property name="title" >
       <string>Output</string>
      </property>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="0" column="1" >
        <widget class="QPushButton" name="outputBrowse" >
         <property name="text" >
          <string>&amp;Browse...</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" >
        <widget class="QLineEdit" name="output" />
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="2" >
     <widget class="QPushButton" name="processButton" >
      <property name="text" >
       <string>Process file</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2" >
     <spacer>
      <property name="orientation" >
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" >
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>467</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar" />
 </widget>
 <resources/>
 <connections/>
</ui>
